doc/releases/pacific.rst

```diff
@@ -2,6 +2,339 @@
 Pacific
 =======
 
+v16.2.8 Pacific   
```

Comment:
@dvanders (from #cephfs): dvanders: around? re `mon_mds_skip_sanity`, we do mention about that in doc/releases/pacific.rst. You wanted to mention that for other major releases (octopus) when upgrading to pacific?